docs/index.asciidoc

```diff
@@ -0,0 +1,209 @@
+:plugin: dissect
+:type: filter
+
+[id="plugins-{type}-{plugin}"]
+
+=== Dissect
+
+include::plugin_information.asciidoc[]
+include::../../../logstash/docs/include/plugin_header.asciidoc[]
```

Comment:
@ph We don't usually commit docs to the logstash-docs repo, but we have a few pages that only exist in logstash-docs, so we do need to commit doc changes occasionally. If we move to one repo, we'll just need to move those files over.

One potential problem with moving all the plugin docs to the logstash repo (I think you're proposing that, right?) is that we'll have a mix of docs that are read-only and updateable. This could be pretty confusing to users/elastic folks who want to submit doc PRs. 

One compromise might be what we do on the Beats team: add a comment to the top of _all_  asciidoc files that are generated so users will know not to update those files. If they do, and submit a PR, we just reject it.  If it's too late for that, maybe we can add that as a future requirement. I'm just concerned that people will be confused about what is updateable if we put everything in the logstash repo.